configuration/pih/scripts/global/zl.js

```diff
@@ -222,4 +222,15 @@ function setupReturnVisitDateValidation(encounterDate, returnVisitDateLessThanEn
 
   jq(domEl).change(returnVisitDateValidation);
   beforeSubmit.push(returnVisitDateValidation);
+  
+
+
+}
+
+function allowNumericalNumber() {
```

Comment:
If we are adding a generic function to global/zl.js, we should make this take the id of the input as a parameter to the allowNumericalNumber function, and not hard-code #receiptNumber here.

Reply:
This isn't really _allow_ a numerical number, but really is intended to _restrict_ to a number, right?  Numbers are already allowed in the text input.  This function is intending to restrict the input to only a number.  So lets name the method accordingly.